app/src/main/res/values/dimens.xml

```diff
@@ -101,6 +101,8 @@
   <dimen name="enter_text_view_padding_start">60dp</dimen>
   <dimen name="enter_text_view_padding_end">60dp</dimen>
   <dimen name="promoted_story_card_width">280dp</dimen>
+  <dimen name="coming_soon_card_width">144dp</dimen>
+  <dimen name="coming_soon_card_width_portrait">160dp</dimen>
```

Comment:
Whats the difference between `coming_soon_card_width ` and `coming_soon_card_width_portrait ` ?

Reply:
I will revert this I am not using it.